website/content/en/docs/ansible/reference/watches.md

```diff
@@ -27,8 +27,7 @@ be monitored for updates and cached.
   current project directory.
 * **vars**: This is an arbitrary map of key-value pairs. The contents will be
   passed as `extra_vars` to the playbook or role specified for this watch.
-* **reconcilePeriod** (optional): The reconciliation interval, how often the
-  role/playbook is run, for a given CR.
+* **reconcilePeriod** (optional): The maximum interval in seconds that the operator will wait before beginning another reconcile, even if no watched events are received. When an operator watches many resources, each reconcile can become expensive, and a low value here can actually reduce performance, so it is generally recommended that this is only used when `watchDependentResources` is `False`. Note that, is recommend to use this option only if your project needs to managing external resources that don't raise Kubernetes events and  only change this value if you know what you are doing.
```

Comment:
How about:

The maximum interval in seconds that the operator will wait before beginning another reconcile, even if no watched events are received. When an operator watches many resources, each reconcile can become expensive, and a low value here can actually reduce performance. Operators that manage resources that are not watched need to use this option, and in that case setting`watchDependentResources` is `False` can help alleviate the performance issues, but that is an advanced use case.

Reply:
I think still not clear the `Note that, is recommend to use this option only if your project needs to managing external resources that don't raise Kubernetes events and only change this value if you know what you are doing.` Could you help improve test make the above more implicity?